crates/bench/src/main.rs

```diff
@@ -56,13 +56,13 @@ fn main() -> ResultBench<()> {
 
     match cli.command {
         Commands::Insert { rows } => {
-            bench_fn!(cli, insert_tx_per_row, rows.unwrap_or(Runs::Tiny), false)
+            bench_fn!(cli, insert_tx_per_row, rows.unwrap_or(Runs::Tiny), false, true)
```

Comment:
Yeah, definitely would like a little less boolean blindness in these call sites.